src/base/string.h

```diff
@@ -138,4 +138,9 @@ inline std::string format_l(const std::string& format_string, Args... args) {
 	return format_impl::format(true, format_string, args...);
 }
 
+/** Query the format tree for the number of required arguments to a format string */
```

Comment:
Format tree? Not something a developer reading this comment should have to concern themselves with ;)

```suggestion
/**
 * Count the required arguments to a format string.
 * @param format_string String to analyze.
 * @return Number of formatting placeholders.
 */
```